tensorboard/components/tf_backend/requestManager.ts

```diff
@@ -156,6 +156,32 @@ export class RequestManager {
     return promise;
   }
 
+  public fetch(url: string, fetchOptions?: object): Promise<Response> {
```

Comment:
Can you use `fetchOptions?: RequestInit`?

Reply:
Absolutely! Did not know about that type. One day, I will learn all these types!